Makefile

```diff
@@ -1,8 +1,13 @@
+SHELL := /bin/bash
+
+test:
+	python -m unittest discover
+
 zip_lambda:
-	zip -j lambda.zip bin/*.py
+	zip -r lambda.zip bin
```

Comment:
Zip into a structure:
```
- bin
--file1.py
--file2.py
```

Instead of:
```
-file1.py
-file2.py
```
This change was made due to https://gist.github.com/gene1wood/06a64ba80cf3fe886053f0ca6d375bc0 - relative imports not working on Lambda